Makefile

```diff
@@ -41,7 +41,7 @@ test-doctests:
 
 .PHONY: test-integration
 test-integration:
-	cargo test --tests -- --test-threads=2
+	cargo test --tests -- --test-threads=1
```

Comment:
I had to reduce the test concurrency further. See https://github.com/momentohq/client-sdk-rust/issues/238

Reply:
I think that issue will be fixed by https://github.com/momentohq/client-sdk-rust/pull/246